README.md

```diff
@@ -2,17 +2,15 @@
 
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/63/badge)](https://bestpractices.coreinfrastructure.org/projects/63)
 [![Coverity passed](https://scan.coverity.com/projects/curl/badge.svg)](https://scan.coverity.com/projects/curl)
-[![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/l1vv31029huhf4g4?svg=true)](https://ci.appveyor.com/project/curlorg/curl)
-[![Azure DevOps Build Status](https://dev.azure.com/daniel0244/curl/_apis/build/status/curl.curl?branchName=master)](https://dev.azure.com/daniel0244/curl/_build/latest?definitionId=1&branchName=master)
```

Comment:
Why remove these?

Reply:
because they present a bad imagine as in they are failing and it looks neater as one line.

Reply:
We could discuss the value of *any* of those silly badges. I figure the point of them is to either provide them all as a way to show project status in several different ways, or we can just remove them and have them replaced with fixed texts instead. The fact that our appveyor and azure CI builds often fail is a known problem we have that we want worked on and solved, not hidden.